### ISSUE TITLE
Torrent files now using hash as name instead of torrent name

### ISSUE THREAD

**Issue by swannie-eire** [transmission/transmission#1501]
Since the update to transmission 3 the torrent files now use the torrent hashes instead of the torrent filenames like in previous version 2.94.

This was seen on Ubuntu 18 bionic release and on FreeBSD 11

**Steps to Dupe**
1). Add a new torrent to transmission.
2). Check the name of the torrent. 
3). You will see the torrent name is the hash and not the file name.

**Current Behaviour**
Torrent files are saved using their hashes.

FreeBSD
```
root@transmission:~ # ls /usr/local/etc/transmission/home/torrents/*
/usr/local/etc/transmission/home/torrents/1111ad5b8421c3227d63fc6a06573dba06b01111.torrent
/usr/local/etc/transmission/home/torrents/1111824711e48e604d457aabcd1320183cc71111.torrent
/usr/local/etc/transmission/home/torrents/111154023d7e9a18ff580ac05b0931ea36cf1111.torrent
```

Ubuntu
```
root@ubuntu:~# ls /var/lib/transmission-daemon/.config/transmission-daemon/torrents/*
/var/lib/transmission-daemon/.config/transmission-daemon/torrents/1111ad5b8421c3227d63fc6a06573dba06b01111.torrent
/var/lib/transmission-daemon/.config/transmission-daemon/torrents/1111824711e48e604d457aabcd1320183cc71111.torrent
/var/lib/transmission-daemon/.config/transmission-daemon/torrents/111154023d7e9a18ff580ac05b0931ea36cf1111.torrent
```

**Expected Behaviour**
Torrent files are saved using their names like in previous versions.

FreeBSD
```
root@transmission:~ # ls /usr/local/etc/transmission/home/torrents/*
/usr/local/etc/transmission/home/torrents/Ubuntu.18.04.torrent
/usr/local/etc/transmission/home/torrents/Ubuntu.16.04.torrent
/usr/local/etc/transmission/home/torrents/Ubuntu.14.04.torrent
```

Ubuntu
```
root@ubuntu:~# ls /var/lib/transmission-daemon/.config/transmission-daemon/torrents/*
/var/lib/transmission-daemon/.config/transmission-daemon/torrents/Ubuntu.18.04.torrent
/var/lib/transmission-daemon/.config/transmission-daemon/torrents/Ubuntu.16.04.torrent
/var/lib/transmission-daemon/.config/transmission-daemon/torrents/Ubuntu.14.04.torrent
```

**Problem**
Makes it very difficult to find torrent file as first you must get the hash.

**Comment by mikedld** [comment]
The change was made intentionally to fix #122. The files in Transmission's directory are for its own use and you shouldn't usually access them directly anyway (or rely on their naming).

**Comment by swannie-eire** [comment]
> The change was made intentionally to fix #122. The files in Transmission's directory are for its own use and you shouldn't usually access them directly anyway (or rely on their naming).

I was afraid you were going to say that. I reverted back to 2.94.

Why i access them directly is that the torrents are added to trans from another machine and this is the only copy of the torrent on the trans server unless i SCP or SFTP a copy up. I upload the torrents on multiple sites and being able to easily identify what torrent is what saves me having to upload another copy to the server for upload to sites.


Thanks for the help
